templates/etc/sudoers.d/ansible.j2

```diff
@@ -1,6 +1,6 @@
-#{{ ansible_managed }}
-
-# Ansible-managed users with NOPASSWD
+###                                     ###
+### This file is controlled by Ansible. ###
+###                                     ###
```

Comment:
Why this change? Isn't it better to keep a variable there? Say we have it defined in our defaults, then we can do fancy things, like print the timestamp of last time this was generated, or anything we like.